ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -98,6 +99,23 @@
             }
         }
 
+        #cacheTabPanelElementIds() {
```

Comment:
Why do we need to cache ids, can't we search them at time of need?

Reply:
Hi @sakshi-arora1 ,
We can find it but as we have already cached the tab and tabPanels, it seemed like a uniform way of doing it.